src/components/LandingPage/Header/index.jsx

```diff
@@ -41,7 +41,7 @@ const Header = () => {
         id="header"
       >
         {/* Header Announcement */}
-        <div className="header__announcement">
+        {/* <div className="header__announcement">
```

Comment:
If you don't need this in the future, can it be removed please?

Reply:
Removed commented-out code.